extensions/crossmodel-lang/src/language-server/cross-model-workspace-manager.ts

```diff
@@ -60,7 +60,7 @@ export class CrossModelWorkspaceManager extends DefaultWorkspaceManager {
          return name !== 'out';
       } else if (entry.isFile) {
          const extname = Utils.extname(entry.uri);
-         return fileExtensions.includes(extname);
+         return selector.fileExtensions.includes(extname);
```

Comment:
```suggestion
         return selector.fileExtensions.includes(extname) || selector.fileNames.includes(Utils.basename(entry.uri));
```

In the newer version Langium also supports selection based on filenames and they added the second check in their code base as well or maybe even call `super.includeEntry` in case it is a file since we do not have any customization there.